src/rp2_common/pico_cyw43_driver/CMakeLists.txt

```diff
@@ -56,28 +56,32 @@ if (EXISTS ${PICO_CYW43_DRIVER_PATH}/${CYW43_DRIVER_TEST_FILE})
             hardware_exception
             )
 
+    if (CYW43_PIN_WL_DYNAMIC)
```

Comment:
@peterharperuk Note that I added this because I noticed it was in https://github.com/raspberrypi/pico-sdk/blob/develop/src/rp2_common/pico_cyw43_driver/include/pico/cyw43_driver.h along with all the other pin-config settings, but it seemed to be missing from https://github.com/raspberrypi/pico-sdk/blob/develop/src/rp2_common/pico_cyw43_driver/CMakeLists.txt

Happy to remove it again if there was a reason it was deliberately omitted!